Source/WPEFramework/PluginServer.cpp

```diff
@@ -250,7 +250,10 @@ namespace PluginHost
 
         _adminLock.Unlock();
 
-        ASSERT(_notifiers.size() == 0);
+        TRACE_L1("Pending notifiers are %lu", _notifiers.size());
```

Comment:
Let's also not add this code as it was not in R2 yet. I only added the unused variable in R3

Reply:
It is just a cleanup, hence added. will remove

Reply:
updated